src/style.css

```diff
@@ -1 +1,86 @@
 /* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: flex;
+  align-self: center;
+  height: 40px;
+}
+
+.nav {
+  box-sizing: border-box;
+}
+
+.logo__img {
+  align-self: center;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  padding-left: 0;
+  margin: 0;
+}
+
+.nav__item {
+  display: flex;
+  align-items: center;
```

Comment:
you are trying to align text vertically, but you have already done so in `.nav__link` by setting `height` and `line-height`

Reply:
Fixed

Reply:
let's figure out why this happens

Reply:
Okay, I think that's because the link is an inline element and so when I set flexbox it becomes block and hence the width gets applied. So I need to add a property of display: block.

Reply:
Got you your point. Fixed now. :)